src/app/server/routes/events.js

```diff
@@ -3,20 +3,24 @@ var router = express.Router();
 var Event = require('../models/Event');
 var passport = require('passport');
 
-
 router.post('/createEvent', function (req, res, next) {
   addToDB(req, res);
 });
 
-
 async function addToDB(req, res) {
 
   var event = new Event({
-    evenTitle: req.body.eventTitle,
+    eventTitle: req.body.eventTitle,
     date: req.body.date,
+    startTime: req.body.startTime,
```

Comment:
When I tested creating an event, it converted it to "am" instead of "pm." Last semester, we ran into this same sort of datetime issue and were converting to our timezone like this:

```
// convert date to CST
function dateCST(date) {
    return moment.utc(date).tz('America/Chicago').format()
}
```

Think we should do this?